src/components/organisms/events/viewer.tsx

```diff
@@ -7,25 +7,18 @@ import { useTranslation } from "react-i18next";
 import { useNavigate, useParams } from "react-router-dom";
 
 import { useEventsDrawer } from "@contexts";
-import { EventsService, LoggerService } from "@services";
-import { dateTimeFormat, namespaces } from "@src/constants";
-import { useToastStore } from "@src/store";
-import { EnrichedEvent } from "@src/types/models";
+import { dateTimeFormat } from "@src/constants";
+import { useEventRedispatch } from "@src/hooks";
 
 import { Frame, Loader, Typography } from "@components/atoms";
 import { IdCopyButton } from "@components/molecules";
 
 export const EventViewer = () => {
-	const [isLoading, setIsLoading] = useState(false);
-	const [eventInfo, setEventInfo] = useState<EnrichedEvent | null>(null);
-
 	const { eventId } = useParams();
-
-	const addToast = useToastStore((state) => state.addToast);
 	const { t } = useTranslation("events", { keyPrefix: "viewer" });
-	const { t: tErrors } = useTranslation("errors");
 	const navigate = useNavigate();
 	const { isDrawer } = useEventsDrawer();
+	const { isLoading, eventInfo, eventInfoError } = useEventRedispatch(eventId);
```

Comment:
If we're getting `eventInfo` from `useEventRedispatch` let's call it: `useEvent` 🤔